src/providers/krb5/krb5_child.c

```diff
@@ -853,7 +853,7 @@ static krb5_error_code sss_krb5_prompter(krb5_context context, void *data,
             }
         }
 
-        DEBUG(SSSDBG_CRIT_FAILURE, "Cannot handle password prompts.\n");
```

Comment:
Hi,

thanks for the patch, `SSSDBG_CRIT_FAILURE` is definitely to low. Maybe you can change the message as well so it does not sound like an error, e.g. `SSSD does not use the krb5 prompter interface for password prompts` or similar?

bye,
Sumit

Reply:
Done.